README.md

```diff
@@ -61,6 +61,8 @@ On **successful** verification of the given pass, the `verifyPassURI` method ret
 {
   "success": true,            // Verification Outcome
   "violates": null,           // Error object if code is invalid
+  "exp": 1652702400,          // Expiration date
```

Comment:
I would recommend calling this `expires` since we're using full words for property names